src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -45,7 +45,7 @@ def __init__(
 
         super().__init__(api_client, config, infra_env_config, nodes)
 
-        self._high_availability_mode = config.high_availability_mode
+        self._control_plane_count = config.control_plane_count
```

Comment:
no need for this line if you set the property below. Then you just need to refer to `self.control_plane_count`.